### PR TITLE
Adding UI modifications

### PR DESCRIPTION
**Pull request by dewmini**

### CODE REVIEW

**Comment by warunalakshitha** [comment]
new .hbs files are missing licence texts.
custom.css file licence text doesn't have the year.

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                                               id="docName" name="docName"
+                                               style="width:100%">
+                                    </div>
+                                </div>
+
+                                <div class="control-group">
+                                    <label class="control-label" for="textarea">Summary</label>
+
+                                    <div class="controls">
+                                        <textarea id="summaryDoc" name="summaryDoc" rows="3"
```
Comment (prasa7) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: Remove inline styles

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                                            <label class="radio" style="margin-left:20px">
+                                                <input type="radio" name="optionsRadios1"
+                                                       id="optionsRadios8"
+                                                       value="file"
+                                                       onclick="javascript:$('#sourceFile').toggle('slow');$('#sourceUrlDoc').hide('slow')">Document
+                                                <a class="icon-question-sign help_popup"
+                                                   data-original-title=""
+                                                   title=""></a>
+
+                                                <p id="fileDoc_help" class="hide">Upload a
```
Comment (prasa7) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: Add multi language support. Correct this in the whole PR

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                            <div style="clear:both; font-size:1px;"></div>
+                        </div>
+                        <br/>
+                        <div class="row-fluid">
+                            <div class="span12">
+                                <div class="control-group">
+                                    <!--<button type="button" class="btn btn-primary" id="saveDocBtn" onclick="saveDoc()">Add Document</button>-->
```
Comment (rmsamitha) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: Remove commented-out code

File: modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  ~ limitations under the License.
+  -->
+<div id="flowChartView" style="display:none">
+    <button class="btn btn-primary" id="flowchartSaveBtn" onclick="saveFlowchart()">Save
+    </button>
+    <button class="btn btn-primary" id="saveAsPNGBtn">Download As PNG</button>
+    <br/><br/>
+
+    <div style="width:100%; height: 10%; white-space:nowrap;">
+                            <span style="display: inline-block; vertical-align: top; padding: 5px; width:15%">
```
Comment (rmsamitha) [inline modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs]: Code formatting issue

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs (hunk 2 above)
Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: This didn't have multi language support.

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs (hunk 1 above)
Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs (hunk 3 above)
Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

File: modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs (hunk 4 above)
Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs]: Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

File: modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                                            <div style="border: 1px solid lightgray; width: 90%; top: 31.5em; position: absolute; left: 0.5em;"></div>
+                                            <div class="window end jsplumb-connected-end" id="endEv"
+                                                 title="To end the flowchart"><strong>
+                                                <p>end</p></strong></div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </span>
+
+                            <span style="display: inline-block; vertical-align: top; padding: 5px; width:85%">
```
Comment (dakshika) [inline modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs]: better to avoid inline styles

Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs]: This is fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

**Pull request merged**